site/en/blog/fugu-showcase/index.md

```diff
@@ -4,6 +4,7 @@ title: Project Fugu API showcase
 subhead: >
   The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in
   the context of Project Fugu.
+description: The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in the context of Project Fugu.
```

Comment:
```suggestion
description: The Project Fugu API Showcase is a collection of apps that use APIs created in the context of Project Fugu.
```

Reply:
This PR is far from being in a reviewable state, but thanks, I’ll happily take this drive-by review.

Reply:
Sorry about that. It looked like it was ready when I came in.